pkg/graph/resolvers/mto_milestone_test.go

```diff
@@ -0,0 +1,43 @@
+package resolvers
+
+import (
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/storage"
+)
+
+func (suite *ResolverSuite) TestCreateMilestoneSolutionLinks() {
```

Comment:
This looks good @OddTomBrooks, Can you also create a test to confirm it works with no solution linked?